recipes/fuc/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "fuc" %}
+{% set version = "0.7.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 9983a82797f914ad4f963bee5eb4ed079ff62400682328c2e52f266a5a993eb0
+
+build:
+  number: 0
+  entry_points:
+    - fuc=fuc.__main__:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: generic
+
+requirements:
+  host:
+    - biopython
+    - lxml
+    - matplotlib-base
+    - numpy
+    - pandas
+    - pip
+    - pyranges
+    - python
+    - seaborn
+  run:
+    - biopython
+    - lxml
+    - matplotlib-base
+    - numpy
+    - pandas
+    - pyranges
+    - python
+    - seaborn
+
+test:
+  imports:
+    - fuc
+    - fuc.api
+    - fuc.cli
+  commands:
+    - fuc --help
+
+about:
+  home: "https://github.com/sbslee/fuc"
+  license: MIT
+  license_family: MIT
+  license_file:
+  summary: "Frequently used commands in bioinformatics"
+  doc_url:
```

Comment:
Mention the ReadTheDocs URL here: https://sbslee-fuc.readthedocs.io/en/latest/

Reply:
Thanks, I added the Read the Docs url.